src/core/public/chrome/ui/header/header_breadcrumbs.tsx

```diff
@@ -73,6 +75,10 @@ export function HeaderBreadcrumbs({
     crumbs = breadcrumbEnricher(crumbs);
   }
 
+  if (dropHomeFromBreadcrumb && crumbs.length && Object.hasOwn(crumbs[0], 'home')) {
```

Comment:
Do we need to check if home is true?

Reply:
we could add this condition as well. There only have one place to set this additional property, i think it's ok only check existence.

Reply:
Can we have the assumption that when `home` is set, it must be true?